source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -246,23 +248,43 @@ def _extend_with_car_park(
     dp_journey.distances.walking += int(walking_speed * car_park_crowfly_duration)
 
 
-def append_path_item_with_access_point(path_items, stop_point, access_point):
+def append_path_item_with_access_point(path_items, stop_point, access_point, language):
+    def get_instruction_words(lan):
+        instructions = {
+            "english_us": u"Then Enter {} via {}.",
+            "french": u"Accéder à {} via {}.",
+            "spanish": u"acceso a {} vía {}.",
+            "italian": u"accesso alla {} vía {}.",
+        }
+        return instructions.get(lan, u"Then Enter {} via {}.")
+
     via = path_items.add()
     via.duration = access_point.traversal_time
     via.length = access_point.length
     via.name = access_point.name
     # Use label in stead of name???
-    via.instruction = u"Then Enter {} via {}.".format(stop_point.label, access_point.name)
+    instruction_words = get_instruction_words(language)
+    via.instruction = instruction_words.format(stop_point.label, access_point.name)
     via.via_uri = access_point.uri
 
 
-def prepend_path_item_with_access_point(path_items, stop_point, access_point):
+def prepend_path_item_with_access_point(path_items, stop_point, access_point, language):
+    def get_instruction_words(lan):
+        instructions = {
+            "english_us": u"Exit {} via {}.",
+            "french": u"Sortir de {} via {}.",
+            "spanish": u"Salida {} vía {}.",
+            "italian": u"Uscita {} via {}.",
+        }
+        return instructions.get(lan, u"Exit {} via {}.")
```

Comment:
Only a suggestion to have the default value be the English version. This will avoid having twice the same string.

```suggestion
        return instructions.get(lan, instructions["english_us"])
```

Reply:
:+1: